### DISCUSSION TITLE
How to differentiate b/w dev mode or build?

### DISCUSSION THREAD

**Discussion by rohit-gohri** [facebook/docusaurus#7621] [Q&A] [answered]
I'm working on https://github.com/rohit-gohri/redocusaurus/issues/195 and for this I need to know when docusaurus is running in dev mode (`start`).

I tired using webpack dev server mode env var `WEBPACK_SERVE` (https://github.com/webpack/webpack-dev-server/pull/3000) but that is not available.

**Comment by Josh-Cena** [comment, marked as answer]
`process.env.NODE_ENV === "production"`. It's documented in a rather obscure place: https://docusaurus.io/docs/next/advanced/ssg#node-env

**Reply by rohit-gohri** [reply]
Of course 🤦 . Seems obvious in hindsight. Thanks!
